docs/docs/getting-started/integration.md

```diff
@@ -80,6 +88,18 @@ Chart.register(
 var myChart = new Chart(ctx, {...});
 ```
 
+A short registration format is also available.
+
+```javascript
+import { Chart, controllers, elements, plugins, scales } from 'chart.js';
+Chart.register(...[
+    ...Object.values(controllers),
+    ...Object.values(elements),
+    ...Object.values(plugins),
+    ...Object.values(scales),
+]);
```

Comment:
We could make this even easier by exporting an *array* containing everything:

```js
import { registrables } from 'chart.js';

Chart.register(...registrables);
```

You might have a better name for `registrables`.